wsidicom/web/wsidicom_web_client.py

```diff
@@ -36,13 +37,18 @@
 
 class WsiDicomWebClient:
     def __init__(
-        self, hostname: str, qido_prefix: str, wado_prefix: str, auth: AuthBase
+        self, hostname: str, qido_prefix: str, wado_prefix: str, auth: Union[AuthBase, Session]
     ):
```

Comment:
There was no doc-string to start with, but perhaps add one now to describe the auth parameter?
The format used elsewhere in the project is:
```python
        """Create a WsiDicomWebSource.

        Parameters
        ----------
        client: WsiDicomWebClient
            Client use for DICOMWeb communication.
        study_uid: Union[str, UID]
            Study UID of DICOM WSI to open.
        series_uid: Union[str, UID]
            Series UID of DICOM WSI top open.
        requested_transfer_syntax: UID
            Transfer syntax to request for image data, for example
            UID("1.2.840.10008.1.2.4.50") for JPEGBaseline8Bit.

        """
```

Reply:
I added a docstring. I also made the arguments optional that are optional for `DICOMwebClient`.